### PR TITLE
#109 template cache mdi.svg instead of request to server to fetch it

### LINKED ISSUE #109: material design icons are not displayed in dropdown
### CURRENT

1. In the sidebar, dropdown `settings-overlay` component don't display material design icons

Capture d’écran de 2020-08-25 15-25-51

1. In the header, dropdown `profile-menu` component don't display material design icons

Capture d’écran de 2020-08-27 11-40-57

### EXPECTED

Capture d’écran de 2020-08-26 15-56-53

Capture d’écran de 2020-08-25 15-28-02

### TO DO

- [ ] display material design icons

### PR DESCRIPTION
**Pull request by tuanlc**

### CODE REVIEW

**Review by itswillta — COMMENTED** [review]
There's a redundant `material.run.js` file.

**Comment by tuanlc** [comment]
> There's a redundant material.run.js file.

Oops! It is needed. I added the content.

File: src/frontend/images/mdi/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Icons
+To use Material Design icons.
+
+- Download 'mdi.svg' from here [1][1].
```
Comment (chamerling) [inline src/frontend/images/mdi/README.md:4]: I do not understand why we need to download?

Comment (tuanlc) [reply src/frontend/images/mdi/README.md:4]: I copy it from the old codebase.

**Review by itswillta — APPROVED** [review]

**Review by chamerling — APPROVED** [review]

**Pull request merged**